lib/travis/listener/app.rb

```diff
@@ -17,15 +17,16 @@ class App < Sinatra::Base
 
       # https://developer.github.com/v3/activity/events/types
       set :events, %w[
-        push
-        pull_request
+        check_run
+        check_suite
         create
         delete
-        repository
         installation
         installation_repositories
-        check_run
-        check_suite
+        member
+        pull_request
+        push
+        repository
```

Comment:
I sorted this list alphabetically, but the only real change is to add `member` as an event type